### DISCUSSION TITLE
Free account disk limit problem

### DISCUSSION THREAD

**Discussion by raphaelMuth** [ravendb/ravendb#17812] [Q&A] [answered]
Hello
I'm trying to solve an issue with RavenDB free instance.
I have almost reached the disk size limit, but even if I remove my databases, or compact them, it doesn't release the disk usage.
Is there another function to clean in the studio, or are the backups using the disk limit?

**Comment by DejanMilicic** [comment, marked as answer]
Please create a support ticket

**Comment by DejanMilicic** [comment]
Also, what you can try to do - you can restart the node. That might clear some temp buffers that might be taking up the space.

**Comment by raphaelMuth** [comment]
I'm using ravendb cloud free.
I dont know if i can restart the node manually, i have searched a way to do so, but i couldn't find it.

I will create support ticket for this. Thank you

**Reply by DejanMilicic** [reply]
Here is how you can do it

On the home screen of Cloud Portal, click on Manage button for your free 1-node cluster

![image](https://github.com/ravendb/ravendb/assets/507133/01e85c4d-c8e5-4010-8157-bb06c073a30a)

Then, scroll down to "Maintenance Zone", click on Unblock and then click on Restart Node

![image](https://github.com/ravendb/ravendb/assets/507133/064258c0-88ef-4377-8b00-6e81b5905da0)

**Reply by raphaelMuth** [reply]
Oh it was not on the studio, i missed this option.
Now i restarted it, but the problem continues.

**Comment by gregolsky** [comment]
Disk space was taken by a soft-deleted database, which remained on the disk, but was not visible in the Studio.

To reclaim disk space in such a case user can re-create a database with the same name and data will be loaded. Afterwards user can hard-delete the database freeing up the space.
